common/unsafe/src/main/java/org/apache/spark/unsafe/Platform.java

```diff
@@ -22,10 +22,14 @@
 import java.lang.reflect.Method;
 import java.nio.ByteBuffer;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import sun.misc.Cleaner;
 import sun.misc.Unsafe;
 
 public final class Platform {
```

Comment:
BTW there is a whole second copy of this class under the 'sketch' module that I think would also have to be modified. Hm, then the new classes need to be copied too, ugh